src/pages/VideoListing.js

```diff
@@ -17,7 +17,7 @@ export function VideoListing() {
   return (
     <div className="explore_main_container">
       {videoList.map((videoItem) => {
-        return <VideoCard data={videoItem} />;
+        return <VideoCard liked={false} data={videoItem} />;
```

Comment:
instead of passing hardcoded value as props you can provide state default value as false. i believe this props is redundant